data/mods/Magiclysm/items/enchanted.json

```diff
@@ -20,25 +20,66 @@
     "type": "GENERIC",
     "symbol": "/",
     "color": "brown",
+    "name": { "str": "minor staff of the magi", "str_pl": "minor staves of the magi" },
+    "description": "A simple carved staff, made of wood.  It faintly glows with magic when you cast spells. Wielding it slightly increases your mana regeneration",
+    "price": 10000,
+    "material": [ "wood" ],
+    "techniques": [ "WBLOCK_2", "RAPID", "SWEEP" ],
+    "flags": [ "SHEATH_SPEAR", "ALWAYS_TWOHAND", "DURABLE_MELEE", "MAGIC_FOCUS" ],
+    "relic_data": {
+      "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "REGEN_MANA", "multiply": 0.5 } ] } ]
+    },
+    "weight": "1400 g",
+    "volume": "3 L",
+    "bashing": 17,
+    "category": "weapons",
+    "to_hit": 3
+  },
+  {
+    "id": "magi_staff_lesser",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "brown",
     "name": { "str": "lesser staff of the magi", "str_pl": "lesser staves of the magi" },
-    "description": "A beautifully carved staff, made of enchanted wood and mithril.  It faintly glows with magic when you cast spells, but it is not a sturdy melee weapon.",
+    "description": "A beautifully carved staff, made of enchanted wood.  It faintly glows with magic when you cast spells. Wielding it increases your mana regeneration",
     "price": 40000,
     "material": [ "wood" ],
-    "flags": [ "SHEATH_SPEAR", "ALWAYS_TWOHAND", "FRAGILE_MELEE", "MAGIC_FOCUS" ],
+    "techniques": [ "WBLOCK_2", "RAPID", "SWEEP" ],
```

Comment:
why did you make this a sturdy melee weapon

Reply:
I took inspiration from the quarterstaff who is more less the same without the magic. Fragile would have made sense if it was a magical orb or a wand.

But I need to admit it might be a bit too strong.

Reply:
Perhaps FRAGILE_MELEE for the minor (weakening from cuts needed to carve runes into it, possibly) and neither DURABLE_MELEE nor FRAGILE_MELEE for the lesser (the increased magic content helps hold it together better)?

Reply:
i see how you could think a quarterstaff is a reasonable thing to base something totally new on, but you changed the item it was before, which was an arcane staff with magical runes carved into it and fragile in melee, to a robust weapon for a martial artist. i would like it to go back to the thing it was, so people actually have to consider the differences in items they use, rather than just "well i'm a melee build so this is a no-brainer"